query-node/substrate-query-framework/cli/src/helpers/ModelCodeGenerator.ts

```diff
@@ -0,0 +1,158 @@
+import { ObjectTypeDefinitionNode, FieldDefinitionNode, ListTypeNode, NamedTypeNode } from 'graphql';
+import { GraphQLSchemaParser } from './SchemaParser';
+
+// Available types for model code generation
+export const availableTypes: { [key: string]: string } = {
+  String: '',
```

Comment:
Is it here possible to introduce your own types, with new names that map to custom Typescript types that you have written yourself? I ask, because this could allow us to introduce support for encodings we get from the runtime, like `u128`, etc.

Reply:
Yes, it is possible. But here again, we face warthog's model generation limitation. Warthog's `codegen` command only support these types for model generation.